metricbeat/module/windows/perfmon/pdh_query_windows_test.go

```diff
@@ -114,3 +115,19 @@ func TestObjectNameRegexp(t *testing.T) {
 		}
 	}
 }
+
+func TestUTF16ToStringArray(t *testing.T) {
+	var array = []string{"\\\\DESKTOP\\Fysieke schijf(0 C:)\\Huidige wachtrijlengte op de schijf", "\\\\DESKTOP\\Fysieke schijf(_Total)\\Percentage schijftijd", ""}
```

Comment:
Do these strings reproduce this problem? Should we introduce some non-ascii character?

Reply:
I have updated the test samples with the problematic values

Reply:
Thanks!